frontend/test/metabase/entities/databases.unit.spec.js

```diff
@@ -0,0 +1,42 @@
+import mock from "xhr-mock";
```

Comment:
Just FYI (no need to change this), all of our similar tests that mock XHR are considered "integration" tests (`*.integ.spec.js`) but honestly the distinction isn't particularly useful.

I think `*.integ.spec.js` should actually use real API calls or at least snapshots of real API calls.